### PR TITLE
View partition documentation and update partition time templates

### LINKED ISSUE #5561: Document the three allowed format strings for custom partitioning
**Update: weekly partitioning is not currently supported.  Only daily, monthly, yearly -- needs to be documented.**  I have filed issues to have unsupported format strings rejected.

Custom partitioning has a big influence on performance.  Once custom partitioning (in Cloud Dedicated/Clustered) is set, the template is unchangeable (short of starting over with a new empty table).  Getting it wrong, but only realizing after investing a lot of time on ingest could be a frustrating experience.  Therefore, best practice is to make sure it's configured correctly before ingesting a lot.  Our documentation should recommend this.

How to verify the template was set (if `influxctl` doesn't already have a way to do this): https://docs.influxdata.com/influxdb/cloud-dedicated/admin/query-system-data/#view-the-partition-template-of-a-specific-table
How to verify that the format string is expanding to as many partitions as expected (as ingest is occurring): https://docs.influxdata.com/influxdb/cloud-dedicated/admin/query-system-data/#view-all-partitions-for-a-table

### Relevant URLs
I suggest adding to at least one of the following:
https://docs.influxdata.com/influxdb/cloud-dedicated/admin/custom-partitions/
https://docs.influxdata.com/influxdb/cloud-dedicated/admin/custom-partitions/best-practices/
https://docs.influxdata.com/influxdb/cloud-dedicated/admin/custom-partitions/define-custom-partitions/
https://docs.influxdata.com/influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/

Existing helpful documentation to cross reference:
https://docs.influxdata.com/influxdb/cloud-dedicated/admin/query-system-data/#view-partition-templates-of-all-tables
https://docs.influxdata.com/influxdb/cloud-dedicated/admin/query-system-data/#view-the-partition-template-of-a-specific-table
https://docs.influxdata.com/influxdb/cloud-dedicated/admin/query-system-data/#view-all-partitions-for-a-table

### What products and version are you using?
Cloud Dedicated (applicable to Clustered too)

### Example use case
I have a dataset with 60000 series, which I wanted to partition into 250 buckets weekly.  I'd never done weekly partitioning before and wasn't feeling confident that I'd gotten the format string right.

```
$ influxctl query --enable-system-tables --database 250p-weekly --token `cat ~/read.token` "SELECT * FROM system.tables"
Queries of InfluxDB system tables may affect production performance while
system tables are accessed. System tables are not part of the stable API
and the schema may change from release to release. [y/n]: y
+------------------------+-------------------------------------------------------------------------------------------+
| table_name             | partition_template                                                                        |
+------------------------+-------------------------------------------------------------------------------------------+
| this_is_my_measurement | {"parts":[{"timeFormat":"%Y-wk%W"},{"bucket":{"tagName":"myKey","numBuckets":250}}]} |
+------------------------+-------------------------------------------------------------------------------------------+
| TOTAL                  | 1 ROWS                                                                                    |
+------------------------+-------------------------------------------------------------------------------------------+
```

But, listing the partitions verified that things looked right.  Note, I had already been running a backfill on the database.

```
$ influxctl query --enable-system-tables --database 250p-weekly --token `cat ~/read.token` "SELECT * FROM system.partitions WHERE table_name = 'this_is_my_measurement' LIMIT 12"
Queries of InfluxDB system tables may affect production performance while
system tables are accessed. System tables are not part of the stable API
and the schema may change from release to release. [y/n]: y
+--------------+------------------------+---------------+--------------------------+-----------+---------------+
| partition_id | table_name             | partition_key | last_new_file_created_at | num_files | total_size_mb |
+--------------+------------------------+---------------+--------------------------+-----------+---------------+
|      1889419 | this_is_my_measurement | 2024-wk17|14  |      1722928811371589748 |         2 |           161 |
|      1889319 | this_is_my_measurement | 2024-wk17|61  |      1722928813158907064 |         2 |           129 |
|      1889226 | this_is_my_measurement | 2024-wk17|50  |      1722928830361329273 |         2 |           138 |
|      1889018 | this_is_my_measurement | 2024-wk04|230 |      1722928844287579208 |         2 |           164 |
|      1890281 | this_is_my_measurement | 2023-wk44|1   |      1722940523432772915 |         2 |           140 |
|      1890294 | this_is_my_measurement | 2023-wk44|117 |      1722940523000622746 |         2 |           123 |
|      1890385 | this_is_my_measurement | 2023-wk44|246 |      1722940519200014945 |         2 |           141 |
|      1896202 | this_is_my_measurement | 2024-wk13|45  |      1723032310937009152 |         2 |           151 |
|      1888957 | this_is_my_measurement | 2024-wk04|52  |      1722928816324541545 |         2 |           163 |
|      1893192 | this_is_my_measurement | 2024-wk09|244 |      1722986437670519611 |         2 |           147 |
|      1889311 | this_is_my_measurement | 2024-wk17|54  |      1722928821006246665 |         2 |           118 |
|      1895114 | this_is_my_measurement | 2024-wk25|39  |      1723020625453961397 |         2 |           136 |
+--------------+------------------------+---------------+--------------------------+-----------+---------------+
|        TOTAL | 12 ROWS                |               |                          |           |               |
+--------------+------------------------+---------------+--------------------------+-----------+---------------+
```

### PR DESCRIPTION
**Pull request by sanderson**
Closes #5561

- Adds information about viewing partition information (@reidkaufmann I left this in here because I think the information is valuable even though the focus of the original issue has changed).
- Updates the supported date specifiers list for partition templates.
- Updates all examples that use unsupported date specifiers when creating custom partitions.

---

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/clustered/reference/cli/influxctl/database/create.md [hunk 1]
```diff
@@ -155,8 +155,8 @@ influxctl database create \
 ### Create a database with a custom partition template
 
 The following example creates a new `mydb` database and applies a partition
-template that partitions by two tags (`room` and `sensor-type`) and by week using
-the time format `%Y wk:%W`:
+template that partitions by two tags (`room` and `sensor-type`) and by day using
+the time format `%d-%m-%Y`:
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/clustered/reference/cli/influxctl/database/create.md:159]: No worries 😄 I just did a bulk find/replace.

File: content/influxdb/cloud-dedicated/admin/custom-partitions/view-partitions.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+- **SQL query**: The SQL query to execute.
+  Pass the query in one of the following ways:
+
+  - a string on the command line
+  - a path to a file that contains the query
+  - a single dash (`-`) to read the query from stdin
+
+{{% code-placeholders "DATABASE_(TOKEN|NAME)|SQL_QUERY" %}}
+
+<!--pytest.mark.skip-->
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/custom-partitions/view-partitions.md:70]: This test should work as-is, without any setup.

File: content/influxdb/cloud-dedicated/admin/custom-partitions/view-partitions.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+| table_name | partition_template                                                                       |
+| :--------- | :--------------------------------------------------------------------------------------- |
+| weather    | {"parts":[{"timeFormat":"%Y-%m-%d"},{"bucket":{"tagName":"location","numBuckets":250}}]} |
+| home       | {"parts":[{"timeFormat":"%Y-%m-%d"},{"tagValue":"room"},{"tagValue":"sensor_id"}]}       |
+| numbers    | {"parts":[{"timeFormat":"%Y"}]}                                                          |
+
+{{% note %}}
+If a table doesn't include a partition template in the output of this command,
+the tables uses the default partition strategy of one day and does not partition
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/custom-partitions/view-partitions.md:130]:
```suggestion
the tables uses the default (1 day) partition strategy and doesn't partition
```

File: content/influxdb/clustered/admin/custom-partitions/view-partitions.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+- **SQL query**: The SQL query to execute.
+  Pass the query in one of the following ways:
+
+  - a string on the command line
+  - a path to a file that contains the query
+  - a single dash (`-`) to read the query from stdin
+
+{{% code-placeholders "DATABASE_(TOKEN|NAME)|SQL_QUERY" %}}
+
+<!--pytest.mark.skip-->
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/custom-partitions/view-partitions.md:70]: This test should run successfully against Cloud Dedicated without any additional setup.

File: content/influxdb/clustered/admin/custom-partitions/view-partitions.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+
+| table_name | partition_template                                                                       |
+| :--------- | :--------------------------------------------------------------------------------------- |
+| weather    | {"parts":[{"timeFormat":"%Y-%m-%d"},{"bucket":{"tagName":"location","numBuckets":250}}]} |
+| home       | {"parts":[{"timeFormat":"%Y-%m-%d"},{"tagValue":"room"},{"tagValue":"sensor_id"}]}       |
+| numbers    | {"parts":[{"timeFormat":"%Y"}]}                                                          |
+
+{{% note %}}
+If a table doesn't include a partition template in the output of this command,
+the tables uses the default partition strategy of one day and does not partition
```
Comment (jstirnaman) [inline content/influxdb/clustered/admin/custom-partitions/view-partitions.md:130]:
```suggestion
the table uses the default (1 day) partition strategy and doesn't partition
```

**Review by jstirnaman — APPROVED** [review]
Small typo fixes ("a tables uses") and minor suggestions.

**Pull request merged**